pyproject.toml

```diff
@@ -1,6 +1,46 @@
+[build-system]
+requires = [
+  "setuptools>=48",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "vllm-spyre"
+description = "vLLM plugin for Spyre hardware support"
+readme = "README.md"
+license = {text = "Apache 2"}
+version = "0.0.0" # TODO dynamic versioning
+dependencies = [
+    "fms-model-optimizer>=0.2.0",
+    "ibm-fms==0.0.8",
+    "vllm",
```

Comment:
We can specify vllm and fms-model-optimizer, and don't have to worry about either of them overriding torch 👍